.github/workflows/deploy-prod-start.yml

```diff
@@ -65,6 +67,40 @@ jobs:
           openshift_token: ${{ env.OPENSHIFT_TOKEN }}
           insecure_skip_tls_verify: true
           namespace: ${{ env.OPENSHIFT_TOOLS_NAMESPACE }}
+      - name: Login to OpenShift Container Registry
```

Comment:
This looks good @areyeslo but I'd like to see this report pre-release (based on the test image)